src/NzbDrone.Core/MediaCover/MediaCoverService.cs

```diff
@@ -109,6 +119,46 @@ private void DownloadCover(Series series, MediaCover cover)
             _httpClient.DownloadFile(cover.Url, fileName);
         }
 
+        private void EnsureResizedCovers(Series series, MediaCover cover, bool forceResize)
+        {
+            switch (cover.CoverType)
+            {
+                default:
+                    break;
+
+                case MediaCoverTypes.Poster:
+                case MediaCoverTypes.Headshot:
+                    EnsureResizedCover(series, cover, 340, null, forceResize);
```

Comment:
we should always worth off of width.

this could get confusing if we do some based on width, some based on height.